S/SpikeSynchrony/Package.toml

```diff
@@ -1,3 +1,3 @@
 name = "SpikeSynchrony"
 uuid = "3713708d-6816-4a57-846e-904b81e02fa1"
-repo = "https://github.com/russelljjarvis/SpikeSynchrony.jl.git"
+repo = "https://github.com/JuliaNeuroscience/SpikeSynchrony.jl"
```

Comment:
You still need the `.git` at the end. It's suppsed to point to a git repository.

Reply:
Okay let me retry.